libs/langchain/langchain/embeddings/nv_aiplay.py

```diff
@@ -0,0 +1,75 @@
+"""Chat Model Components Derived from ChatModel/NVAIPlay"""
+
+import asyncio
+from typing import Any, Dict, List, Literal, Optional
+
+from langchain.llms.nv_aiplay import ClientModel, NVCRModel
+from langchain.pydantic_v1 import Field, root_validator
+from langchain.schema.embeddings import Embeddings
+
+## NOTE: This file should not be ran in isolation as a single-file standalone.
+## Please use llms.nv_aiplay instead.
+
+
+class NVAIPlayEmbeddings(ClientModel, Embeddings):
+    """NVIDIA's AI Playground NVOLVE Question-Answer Asymmetric Model."""
+
+    client: NVCRModel = Field(NVCRModel)
+    model_name: str = Field("nvolveqa")
+    model: Optional[str] = Field(None)
```

Comment:
Awesome! Yeah, I kinda hated that, but was following other implementations' examples. Will fix.

Reply:
Thanks! That was mostly because the initial implementations of some integrations went a different way and we didn't want to make a breaking change by removing entirely